### PR TITLE
SR Studio V 1.2.0 Release

### PR DESCRIPTION
**Pull request by TensAndTwenties**
### Changed

- Upgraded follow-redirects dependency for security alert
- Bumped Node version for tests to 16.x
- Bumped electron version
- Fixed unit test errors
- Created new testing process for CaptionStudio

### Fixed

- fixed bug when selecting a caption that caused the new audio file to not be selected and loaded into the wave form

### CODE REVIEW

File: src/renderer/class/CaptionManager.js [hunk 1]
```diff
@@ -99,7 +100,8 @@ class CaptionManager {
       this.emitOpenedJSON();
       return;
     }
-    this.currentCaptionIndex.edited = true;
+    //this.currentCaptionIndex.edited = true;
```
Comment (902seanryan) [inline src/renderer/class/CaptionManager.js:103]: This is probably leftover from months ago from me lol, can you just remove it

File: src/renderer/components/caption-studio/TextEditor.vue [hunk 2]
```diff
@@ -154,6 +154,7 @@ export default {
     EventBus.$on('caption_reset', this.reset);
     EventBus.$on('json_errors', this.onJsonErrors);
     this.$refs.Quill?.quill?.on('text-change', this.onEdit);
+    //this.$refs.Quill?.$el.addEventListener('paste', catchPasteEvent);
```
Comment (902seanryan) [inline src/renderer/components/caption-studio/TextEditor.vue:157]: Another thing that should be removed

File: src/main/index.js [hunk 3]
```diff
@@ -43,9 +44,11 @@ function createWindow() {
     webPreferences: {
       // Use pluginOptions.nodeIntegration, leave this alone
       // See nklayman.github.io/vue-cli-plugin-electron-builder/guide/security.html#node-integration for more info
-      nodeIntegration: process.env.ELECTRON_NODE_INTEGRATION,
+      nodeIntegration: true,
```
Comment (902seanryan) [inline src/main/index.js:47]: this should probably be switched back to the previous `process.env.ELECTRON_NODE_INTEGRATION` unless it breaks thing in which case we should update the comment

**Review by 902seanryan — CHANGES_REQUESTED** [review]
just a couple small things (which are mostly my fault anyway)

File: src/renderer/components/caption-studio/TextEditor.vue (hunk 2 above)
Comment (deycorinne) [inline src/renderer/components/caption-studio/TextEditor.vue:157]: its fine now bc it already made it to the release, but in the future we should make sure to remove commented code like this or at least leave a comment explaining why it is still there!

**Review by deycorinne — APPROVED** [review]

**Review by 902seanryan — APPROVED** [review]

File: src/renderer/components/caption-studio/TextEditor.vue (hunk 2 above)
Comment (TensAndTwenties) [reply src/renderer/components/caption-studio/TextEditor.vue:157]: I think this line was something we wanted to come back to and evaluate later. It was causing errors, but we still want to find another way to add the event listener - there may already be a ticket about it. Does this ring a bell @902seanryan ? 

Either way I will remove the comment, just wanted to call that out!

Comment (902seanryan) [reply src/renderer/components/caption-studio/TextEditor.vue:157]: That sounds right

**Pull request merged**